hackathon_site/event/static/event/js/script.js

```diff
@@ -0,0 +1,63 @@
+// Change navbar color on scroll
+$(document).scroll(function () {
+    let $nav = $(".navbar");
+    $nav.toggleClass("scrolled", $(this).scrollTop() > $nav.height());
+});
+
+$(document).ready(function () {
+    // Materialize stuff
+    $(".sidenav").sidenav();
+    $(".carousel").carousel({ dist: 0, padding: 600 });
+    setInterval(function () {
+        $(".carousel").carousel("next");
+    }, 3000);
+
+    $(".scrollspy").scrollSpy();
+    $(".collapsible").collapsible();
+
+    // Countdown stuff
+
+    // Registration open and close date, and event date
+    let regOpenDate = new Date("Sep 1, 2020 00:00:00").getTime();
+    let regCloseDate = new Date("Sep 30, 2020 00:00:00").getTime();
+    let eventDate = new Date("Oct 31, 2020 00:00:00").getTime();
+
+    let now = new Date().getTime();
+
+    // Set the title based off what it's counting down to
+    if (regOpenDate > now) {
+        countDownDate = regOpenDate;
+        $("#countdownTitle").html("Registration Opens In");
+    } else if (regCloseDate > now) {
+        countDownDate = regCloseDate;
+        $("#countdownTitle").html("Registration Closes In");
+    } else if (eventDate > now) {
+        countDownDate = eventDate;
+        $("#countdownTitle").html("Event Starts In");
+    }
+
+    // Delete the entire countdown if event start date has passed
```

Comment:
This can live here for now, but eventually this is something that would be better to happen at the template level rather than in JS